staging/src/kubevirt.io/client-go/go.mod

```diff
@@ -5,100 +5,98 @@ go 1.17
 require (
 	github.com/coreos/prometheus-operator v0.38.0
 	github.com/go-kit/kit v0.9.0
-	github.com/go-openapi/spec v0.19.3
 	github.com/golang/glog v1.0.0
-	github.com/golang/mock v1.5.0
+	github.com/golang/mock v1.6.0
 	github.com/google/gofuzz v1.1.0
 	github.com/gorilla/websocket v1.5.0
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v0.0.0-20191119172530-79f836b90111
 	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.2.0
-	github.com/onsi/ginkgo/v2 v2.1.3
-	github.com/onsi/gomega v1.18.1
+	github.com/onsi/ginkgo/v2 v2.4.0
+	github.com/onsi/gomega v1.23.0
 	github.com/openshift/client-go v0.0.0-20210112165513-ebc401615f47
 	github.com/spf13/pflag v1.0.5
-	k8s.io/api v0.23.5
-	k8s.io/apiextensions-apiserver v0.23.5
-	k8s.io/apimachinery v0.23.5
+	k8s.io/api v0.26.3
+	k8s.io/apiextensions-apiserver v0.26.3
+	k8s.io/apimachinery v0.26.3
 	k8s.io/client-go v12.0.0+incompatible
```

Comment:
Shouldn't this be bumped to v0.26.3 as well?

Reply:
It is bumped in the replace section below. I can bump it here too (and will do, thanks), but from a functionality perspective it is ok.